lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -148,18 +147,80 @@ function dosomething_campaign_preprocess_common_vars(&$vars, &$wrapper) {
     $vars['share_bar'] = dosomething_helpers_share_bar($campaign_path, $share_types, 'problem_shares', 'js-analytics-problem', 'social-share-bar -with-callout');
   }
 
+  // Hot and win module variables.
+  if (dosomething_campaign_is_win_module($vars) || dosomething_campaign_is_hot_module($vars)) {
+    $author = user_load($vars['uid']);
+    $first_name = check_plain($wrapper->author->field_first_name->value());
+    $last_name = check_plain($wrapper->author->field_last_name->value());
+
+    $vars['author_name']  = $first_name . ' ' . $last_name;
+    $vars['author_title'] = check_plain($wrapper->author->field_job_title->value());
+    if($author->picture) {
+      $vars['author_image'] = dosomething_image_get_themed_image_by_fid($author->picture->fid, '300x300');
+    }
+
+    $vars['reportback_noun_verb'] = $wrapper->field_reportback_noun->value() . " " . $wrapper->field_reportback_verb->value();
+  }
 }
 
 /**
- * Preprocess variables for win module.
+ * Create the markup for the win module.
  */
-function dosomething_campaign_preprocess_win_module(&$vars, &$wrapper) {
+function dosomething_campaign_preprocess_win_module(&$vars, $wrapper) {
   $node = node_load($vars['nid']);
-  $vars['share_copy'] = variable_get('dosomething_campaign_win_share_copy');
   $rb_progress = dosomething_reportback_get_reportback_total_plus_override($node->nid);
   $copy_token_replaced = token_replace(variable_get('dosomething_campaign_win_copy'), array('node' => $node));
-  $vars['win_copy'] = str_replace('[campaign_progress]', $rb_progress, $copy_token_replaced);
+
+  // Set background image with inline CSS.
   $background_fid = variable_get('dosomething_campaign_win_background_image');
+  $file = file_load($background_fid);
+  $background_image_uri = file_create_url($file->uri);
+
+  $win_module_image = '.win-module__progress {
+      background-image: url("' . $background_image_uri . '");
+    }
+
+    @media only screen and (min-width: 768px) {
+      .win-module__progress {
+        background-image: url("' . $background_image_uri . '");
+      }
+    }';
+  drupal_add_css($win_module_image, $option['type'] = 'inline');
+
+  // Share bar
+  // @TODO - Copy is placeholder
+  $share_types = array(
+    'facebook' => array(
+      'type' => 'feed_dialog',
+      'parameters' => array(
+        'picture' => $vars['hero_image']['desktop'],
+        'caption' => "We rocked it. Let’s keep it up!",
+      ),
+    ),
+    'twitter' => array(
+      'tweet' => "We rocked it. Let’s keep it up!",
+    ),
+    'tumblr' => array(
+      'posttype' => 'photo',
+      'content' => $vars['hero_image']['desktop'],
+      'caption' => "We rocked it. Let’s keep it up!",
```

Comment:
want to just set this text as generic share text var since it's the same across all three?